source/Patches/Utils.cs

```diff
@@ -447,7 +447,17 @@ public static void MurderPlayer(PlayerControl killer, PlayerControl target)
                     target.myTasks.Insert(0, importantTextTask);
                 }
 
-                killer.MyPhysics.StartCoroutine(killer.KillAnimations.Random().CoPerformKill(killer, target));
+                // perform kill without body.
+                //TODO: find a better way to perform this kill
+                if (
```

Comment:
I think I'm missing something, why did this need to change?

Reply:
the principle of this option is that if a lover gets a vote and the second must die, then he will die without there being a body, no body, no postponement

Reply:
I understand. Why did this specific code change get made?